lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7471,6 +7471,75 @@ bool importer::isForeignReferenceTypeWithoutImmortalAttrs(const clang::QualType
          !hasImmortalAtts(pointeeType->getDecl());
 }
 
+llvm::SmallVector<const clang::RecordDecl *, 4>
+getMatchingParentDecls(const clang::RecordDecl *decl) {
```

Comment:
Now that this only returns records that have the import as reference attribute, I think the name of this function should reflect this.